src/SourceBuild/content/eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -17,6 +17,8 @@ resources:
     trigger:
       branches:
         include:
+          - refs/heads/release/9.0.1xx
+          - refs/heads/release/9.0.1xx-rc*

```

Comment:
```suggestion
          - refs/heads/release/*.0.1xx
          - refs/heads/release/*.0.1xx-rc*
```

Reply:
Why not just `refs/heads/release/*.0.1xx*`?  Release branches are protected, they should not be created as dev 1-off branches.  My concern about not having the * is for real 1-off releases.  They would normally would be in internal only but what if ...

Reply:
Yep, that makes sense.

Reply:
So I will keep `9.0.1xx` instead of `*.0.1xx` for temporary. If they moved these branches to `internal/release`, I will delete this change. @MichaelSimons @mthalman Would this be feasible?